### PR TITLE
feat: update subscriptions. Add TonWallet test. Add TokenWallet test

### PR DESCRIPTION
**Pull request by Alex-A4**
## Status

**READY**

## Description

- Bump bridge version
- Update `TonWalletRepository.updateSubscription` and `TokenWalletRepository.updateTokenSubscriptions` as it was started in #44 
- Add tests to `TonWalletRepository` for subscsriptions
- Add tests to `TokenWalletRepository` for subscriptions

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [X] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/src/repositories/refresh_polling_queue/cancellable_operation_awaited.dart [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  final CancelableOperation<T> _operation;
+
+  bool _isCanceled = false;
+
+  bool get isCanceled => _isCanceled;
+
+  Future<void> cancel() {
+    _isCanceled = true;
+    return _operation.valueOrCancellation();
```
Comment (nesquikm) [inline lib/src/repositories/refresh_polling_queue/cancellable_operation_awaited.dart:17]: I don't get it: you don't call super.cancel() at all?

File: lib/src/repositories/refresh_polling_queue/cancellable_operation_awaited.dart [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+/// Wrapper above CancelableOperation that allows mark operation as canceled
+/// and wait for canceled operation itself except of ignoring result.
+class CancelableOperationAwaited<T> {
+  CancelableOperationAwaited.fromFuture(Future<T> result)
+      : _operation = CancelableOperation.fromFuture(result);
+
+  final CancelableOperation<T> _operation;
+
+  bool _isCanceled = false;
```
Comment (nesquikm) [inline lib/src/repositories/refresh_polling_queue/cancellable_operation_awaited.dart:11]: Typo: cance**ll**ed (double l)

Comment (Alex-A4) [reply lib/src/repositories/refresh_polling_queue/cancellable_operation_awaited.dart:11]: Copied from original class :)

File: lib/src/repositories/refresh_polling_queue/cancellable_operation_awaited.dart (hunk 1 above)
Comment (nesquikm) [reply lib/src/repositories/refresh_polling_queue/cancellable_operation_awaited.dart:17]: Okay, but what is the point of use this library then? valueOrCancellation() will wait for super's cancellation, isn't it?

**Review by nesquikm — APPROVED** [review]

**Pull request merged**